### PR TITLE
[esw, teleop] science, arm, drive, general integration

### PR DESCRIPTION
**Pull request by cgiger00**

### CODE REVIEW

File: base_station/gui/src/components/ERDRoverMap.vue [hunk 1]
```diff
@@ -6,9 +6,18 @@
       <l-tile-layer :url="this.online ? this.onlineUrl : this.offlineUrl" :attribution="attribution" :options="tileLayerOptions"/>
       <l-marker ref="rover" :lat-lng="odomLatLng" :icon="locationIcon"/>
 
-      <l-marker :lat-lng="waypoint.latLng" :icon="waypointIcon" v-for="waypoint, index in waypointList" :key="index">
-        <l-tooltip :options="{ permanent: 'true', direction: 'top'}"> {{ waypoint.name }}, {{ index }} </l-tooltip>
-      </l-marker>
+      <div v-for="(waypoint, index) in waypointList" :key="index">
+        <div v-if="index===highlightedWaypoint">
```
Comment (cgiger00) [inline base_station/gui/src/components/ERDRoverMap.vue:10]: @jnnanni is this now missing from the AutonRoverMap.vue file?

File: ansible/roles/base_station_services/files/rover-base_station_gstreamer_3.service [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Unit]
```
Comment (tabiosg) [inline ansible/roles/base_station_services/files/rover-base_station_gstreamer_3.service:1]: ill delete the extra gstreamer services

**Review by tabiosg — APPROVED** [review]

File: ansible/roles/base_station_services/files/rover-base_station_gstreamer_3.service (hunk 2 above)
Comment (cgiger00) [reply ansible/roles/base_station_services/files/rover-base_station_gstreamer_3.service]: right forgot about that

**Review by tabiosg — CHANGES_REQUESTED** [review]
Jetson cameras doesn't pass linter... I thought I tested this :(( I'll make a pr tomorrow morning (?)

**Comment by cgiger00** [comment]
sad

**Review by tabiosg — APPROVED** [review]

**Review by sashreek1 — APPROVED** [review]

**Pull request merged**